test/fixtures/compound17.css

```diff
@@ -0,0 +1,49 @@
+
+:root
+{
+	--color: #ffffff;
+}
+
+#foo1 ul > li {
+	color: var(--color);
+}
+
+#foo2 ul ~ li {
+	color: var(--color);
+}
+
+#foo3 ul + li {
+	color: var(--color);
+}
```

Comment:
I think all of the `#foo1`, `#foo2`, and `#foo3` rules can be split into their own tests